dwigradcheck doesn't respect -force flag

We are running dwigradcheck command with -force flag to override already existing output file.

```
dwigradcheck raw1.mif -grad raw1.b -mask ${mask}.mif -export_grad_mrtrix cor1.b -tempdir ./tmp -force -nthreads $NCORE -quiet

```

However, this command fails if the output file (cor1.b in this case) already exist.

```
...
dwigradcheck: [ERROR] Command failed: mrinfo data.mif -grad grad_flipnone_perm012_scanner.b -export_grad_mrtrix /mnt/disks/sec/prod/5d2e1b4278d9fa1cee2aa79d/5d2e89cb78d9fa1cee2af416/cor1.b (dwigradcheck:212)
dwigradcheck: Output of failed command:
              mrinfo: [ERROR] output file "/mnt/disks/sec/prod/5d2e1b4278d9fa1cee2aa79d/5d2e89cb78d9fa1cee2af416/cor1.b" already exists (use -force option to force overwrite)
dwigradcheck: Script failed while executing the command: mrinfo data.mif -grad grad_flipnone_perm012_scanner.b -export_grad_mrtrix /mnt/disks/sec/prod/5d2e1b4278d9fa1cee2aa79d/5d2e89cb78d9fa1cee2af416/cor1.b
dwigradcheck: For debugging, inspect contents of temporary directory: /mnt/disks/sec/prod/5d2e1b4278d9fa1cee2aa79d/5d2e89cb78d9fa1cee2af416/tmp/dwigradcheck-tmp-BEUH1D/
```

--- a/bin/dwigradcheck
+++ b/bin/dwigradcheck
@@ -209,7 +209,7 @@ if grad_export_option:
     grad_import_option = ' -grad grad' + suffix + '.b'
   elif best[3] == 'image':
     grad_import_option = ' -fslgrad bvecs' + suffix + ' bvals'
-  run.command('mrinfo data.mif' + grad_import_option + grad_export_option)
+  run.command('mrinfo data.mif' + grad_import_option + grad_export_option + (' -force' if app.forceOverwrite else ''))
 
 
 app.complete()

